micropub.php

```diff
@@ -200,34 +209,34 @@ private static function authorize() {
 	public static function post_handler( $user_id ) {
 		$status = 200;
 		$action = static::$input['action'];
-		$url = static::$input['url'];
+		$url    = static::$input['url'];
 
 		// check that we support all requested syndication targets
 		$synd_supported = apply_filters( 'micropub_syndicate-to', array(), $user_id );
 		$synd_requested = static::$input['properties']['syndicate-to'];
-		$unknown = array_diff( $synd_requested, $synd_supported );
+		$unknown        = array_diff( $synd_requested, $synd_supported );
 
 		if ( $unknown ) {
-			static::error( 400, 'Unknown syndicate-to targets: ' . implode(', ', $unknown ) );
+			static::error( 400, 'Unknown syndicate-to targets: ' . implode( ', ', $unknown ) );
 
-		// create
-		} elseif ( ! $url || $action == 'create' ) {
+			// create
```

Comment:
this is wrong. same with the rest of the comment indents below. it should either be where it was before, or if that's impossible, append it to the line below.